sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/AppConfigurationSnapshotCreatedEventData.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.messaging.eventgrid.systemevents;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.AppConfiguration.SnapshotCreated event. */
+@Fluent
+public final class AppConfigurationSnapshotCreatedEventData {
+    /*
+     * The name of snapshot that was created.
+     */
+    @JsonProperty(value = "name")
+    private String name;
+
+    /*
+     * The etag representing the new state of the snapshot.
+     */
+    @JsonProperty(value = "etag")
+    private String etag;
+
+    /*
+     * The sync token representing the server state after the event.
+     */
+    @JsonProperty(value = "syncToken")
+    private String syncToken;
+
+    /** Creates an instance of AppConfigurationSnapshotCreatedEventData class. */
+    public AppConfigurationSnapshotCreatedEventData() {}
+
+    /**
+     * Get the name property: The name of snapshot that was created.
```

Comment:
nit: should be `The name of the snapshot that was created.`

Reply:
good catch!

Reply:
Getting a swagger update for this one.